docs/datasources/core/ipsec_connection_device_status.md

```diff
@@ -1,6 +1,10 @@
 # oci\_core\_ipsec\_status
 
-Gets a list of internet gateways.
+**API:** [IPSecConnectionDeviceStatus Reference][6c32ac75]
+
+  [6c32ac75]: https://docs.us-phoenix-1.oraclecloud.com/api/#/en/iaas/20160918/IPSecConnectionDeviceStatus/ "IPSecConnectionDeviceStatusReference"
+
+~~Gets a list of internet gateways.~~ Gets information about the status of the IPSec connection.
```

Comment:
Should "Gets a list of internet gateways." be removed?

Reply:
I think so, it's not accurate